givitsite/templates/request.html

```diff
@@ -6,15 +6,37 @@ <h1>How can we help?</h1>
 <form action="" method="POST">
 
     {%csrf_token%}
-    what item do you need?: <input type = "text" name = "item" />
-    <br>
-    city to deliver: <input type = "text" name = "city" />
-    <br>
+    what item do you need?: 
+    <select id="item" name="item" size="1">
+      <option value="bed">Bed</option>
+      <option value="desk">desk</option>
+      <option value="TV">TV</option>
+      <option value="washing machine">washing machine</option>
+      <option value="bed">laptop</option>
+      <option value="fridge">fridge</option>
+      <option value="oven">oven</option>
+      <option value="sofa">sofa</option>
+      <option value="microwave oven">microwave oven</option>
+      <option value="chair">chair</option>
+      <option value="computer screen">computer screen</option>
+      <option value="keyboard">keyboard</option>
```

Comment:
Suggestion: maybe it is a good idea to save these options in the database? Same goes for cities.

Reply:
we will definitely do it. I'll post a new issue for this gap.